CONTRIBUTING.md

```diff
@@ -55,9 +56,72 @@ If it seems like no one is working on the issue, great! You've officially found
 
 ### Creating a pull request
 
+Once you're happy with your changes, commit them:
+
+- `git add .`
+- `git commit -m "your commit message"`
+- `git push origin your-branch` (where `your-branch` is the name of the branch you created at the beginning). Yhis will push your changes to `your-branch` of your own fork of the `summer-of-code` repo on github)
+- make a pull request by navigating to your repo (`http://www.github.com/your_username/summer-of-code`) and your branch (`your-branch`), and by clicking the "Compare & Review" button, as explained [here](https://help.github.com/articles/using-pull-requests/).
+
 You can find very thorough information on Pull Requests in [GitHub's documentation](https://help.github.com/articles/creating-a-pull-request/).
 The most important thing to keep in mind is that all pull requests should be made against `gh-pages`, which is our main branch.
 
```

Comment:
How about adding a screenshot? This part was quite puzzling to me when I just started with PR's.